src/browser/screen.js

```diff
@@ -6,7 +6,7 @@
  *
  * @param {BusConnector} bus
  */
-function ScreenAdapter(screen_container, bus)
+function ScreenAdapter(screen_container, bus, screen_options = undefined)
```

Comment:
I think the code would be a bit simpler if you force this to be an object.

Reply:
Patched.